source/assets/css/style.css

```diff
@@ -916,10 +916,6 @@ video {
   margin-bottom: 36px;
```

Comment:
If you exclude this file now, it should be removed too 😄

Reply:
This bugger is tenacious. I had deleted it! But then switched to another branch, stash popped and there it was again. It didn't see that it said modified, not deleted.